tedana/workflows/tedana.py

```diff
@@ -367,6 +378,8 @@ def tedana_workflow(
     prefix : :obj:`str` or None, optional
         Prefix for filenames generated.
         Default is ""
+    masktype : {'dropout', 'decay'} or :obj:`list`, optional
```

Comment:
```suggestion
    masktype : :obj:`list` with 'dropout' and/or 'decay' or None, optional
```
I'm fairly sure the function requires a list, even for a single element so i reworded the docstring.